Resources/Audio/DeltaV/Jukebox/attributions.yml

```
@@ -0,0 +1,77 @@
+  source: "https://masterbootrecord.bandcamp.com/album/c-edit-config-sys"
+
+- files: ["drozerix_-_alone.xm-MONO.ogg"]
+  license: "Custom"
+  copyright: "Alone by Drozerix, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=199968"
+
+- files: ["drozerix_-_leisurely_voice.xm-MONO.ogg"]
+  license: "Custom"
+  copyright: "Leisurely Voice by Drozerix, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=183837"
+
+- files: ["femtanyl_-_MURDER_EVERY_1_U_KNOW_feat._takihasdied_MONO.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "MURDER EVERY 1 U KNOW by Femtanyl featuring takihasdied, used with the understanding that this is a noncommercial project with credit given to the author, also converted to mono."
+  source: "https://femtanyl.bandcamp.com/track/murder-every-1-u-know-feat-takihasdied"
+
+- files: ["hackers-MONO.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Hackers by Karl Casey @ White Bat Audio, converted to mono"
+  source: "https://www.youtube.com/watch?v=k8nHWwO1U2Q"
+
+- files: ["marhaba-MONO.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Marhaba by Ian Alex Mac. Converted from MP3 to OGG, then converted to mono"
+  source: "https://freemusicarchive.org/music/Ian_Alex_Mac/Cues/Marhaba"
+
+- files: ["Patricia_Taxxon_-_Minute_-_MONO.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Minute by Patricia Taxxon off the album 'Aeroplane,' converted to mono"
+  source: "https://patriciataxxon.bandcamp.com/track/minute"
+
+- files: ["psirius_-_nymphs_of_the_forest.mptm-MONO.ogg"]
+  license: "CC-BY-NC-SA-4.0"
+  copyright: "Nymphs of the forest by Psirius, converted to mono"
+  source: "https://modarchive.org/index.php?request=view_by_moduleid&query=185545"
+
+- files: ["Scratch_Post_-_OST_MONO.ogg"]
+  license: "CC-BY-SA-4.0"
+  copyright: "Scratch Post by Ghirardelli7 on SoundCloud, used with the understanding that credit is given to the artist."
+  source: "https://soundcloud.com/ghirardelli7/scratch-post-ost"
+
+- files: ["shibamata-MONO.ogg"]
+  license: "Custom"
+  copyright: "Shibamata by .2gou / Dot Nigou. This track is not released under any formal licensure, and exists in the public domain in multiple forms, including remixes and mashups. Converted to mono."
+  source: "https://www.youtube.com/watch?v=FIw-HUP7XK0"
+
+- files: ["space_asshole.ogg"]
+  license: "Custom"
+  copyright: "Space Asshole by Chris Remo is used with special permission from the author, under the condition that the project remains non-commercial and open source. The author also requested that a link to his bandcamp be included: https://chrisremo.bandcamp.com/ Converted from stereo to mono."
+  source: "https://idlethumbs.bandcamp.com/track/space-asshole"
+
+- files: ["superposition-MONO.ogg"]
+  license: "CC-BY-NC-3.0"
+  copyright: "Superposition by Amie Waters, converted to mono"
+  source: "https://amiewaters.bandcamp.com/track/superposition-2"
```

```suggestion
  source: "https://amiewaters.bandcamp.com/track/superposition-2"

```
line at the end is good form